sycl/source/detail/pi.cpp

```diff
@@ -603,6 +603,7 @@ void DeviceBinaryImage::init(pi_device_binary Bin) {
                                __SYCL_PI_PROPERTY_SET_COMPOSITE_SPEC_CONST_MAP);
   DeviceLibReqMask.init(Bin, __SYCL_PI_PROPERTY_SET_DEVICELIB_REQ_MASK);
   KernelParamOptInfo.init(Bin, __SYCL_PI_PROPERTY_SET_KERNEL_PARAM_OPT_INFO);
+  SyclIsEsimdImage.init(Bin, __SYCL_PI_PROPERTY_SET_SYCL_IS_ESIMD_IMAGE);
```

Comment:
Is it possible to write a unit test (similar to https://github.com/intel/llvm/blob/sycl/sycl/unittests/kernel-and-program/KernelRelease.cpp) which checks if the new property is recognized?

Reply:
Seems that it is not that easy. In order to test this code, we need to construct an in-memory object of  `pi_device_binary_struct`, which then would be parsed by `DeviceBinaryImage` class. The `clang-offload-wrapper` tool seems to do similar things, but there is no code we can directly reuse.

Reply:
@kbobrovs, the patch is approved by code owners.
Could you mark this conversation as resolved if your comment is addressed, please?